V7_MIGRATION.md

```diff
@@ -7,13 +7,26 @@ _Documentation for v7 will be published to https://developer.paypal.com/braintre
 ## Table of Contents
 
 1. [Supported Versions](#supported-versions)
+1. [Card](#card)
 1. [Venmo](#venmo)
 1. [SEPA Direct Debit](#sepa-direct-debit)
 
 ## Supported Versions
 
 v7 bumps to a minimum deployment target of iOS 16+.
 
+## Card
+v7 updates `BTCard` to require setting all properties through the initializer, removing support for dot syntax. To construct a `BTCard`, pass the properties directly in the initializer:
+
+```
+let card = BTCard(
+    number = "4111111111111111",
+    expirationMonth = "12",
+    expirationYear = "2025",
+    cvv = "123"
+)
```

Comment:
Take it or leave it - I wonder if we even need a code sample for this? It's a pretty self explanatory change to merchants that we call out in our CHANGELOG. Just thinking that the migration guide will get pretty long since every request object will have this change.

What do you think?

Reply:
You're right, updated: [c127063](https://github.com/braintree/braintree_ios/pull/1443/commits/c127063a846a995a4f9b039521ab7d5f84d8663d)